app/vminsert/relabel/relabel.go

```diff
@@ -49,7 +51,17 @@ func Init() {
 	configTimestamp.Set(fasttime.UnixTimestamp())
 
 	go func() {
-		for range sighupCh {
+		var tickerCh <-chan time.Time
+		if *relabelConfigCheckInterval > 0 {
+			ticker := time.NewTicker(*relabelConfigCheckInterval)
+			tickerCh = ticker.C
+			defer ticker.Stop()
+		}
+		for {
+			select {
+			case <-tickerCh:
+			case <-sighupCh:
+			}
 			configReloads.Inc()
 			logger.Infof("received SIGHUP; reloading -relabelConfig=%q...", *relabelConfig)
```

Comment:
Just removing reference to `SIGHUP` since config could be reloaded due to periodic reload. This will make the message consistent with https://github.com/VictoriaMetrics/VictoriaMetrics/blob/b98e5927525ceb3238870701ea9c7dcf6296681d/app/vmagent/remotewrite/relabel.go#L76
```suggestion
			logger.Infof("reloading relabel configs pointed by  -relabelConfig=%q...", *relabelConfig)
```